components/github-api-client/src/config.rs

```diff
@@ -42,15 +41,15 @@ pub struct GitHubCfg {
     /// Client secret used for GitHub API requests
     pub client_secret: String,
     /// App Id used for builder integration
-    pub app_id: u64,
+    pub app_id: u32,
 }
 
 impl Default for GitHubCfg {
     fn default() -> Self {
         GitHubCfg {
             url: DEFAULT_GITHUB_URL.to_string(),
             web_url: DEFAULT_GITHUB_WEB_URL.to_string(),
-            app_private_key: "".to_string(),
+            app_private_key: "/src/.secrets/habitat-builder-dev.pem".to_string(),
```

Comment:
This default is basically for us running in our local dev environments using the right key, correct?

If so, adding that information to `BUILDER_DEV.md` would be nice.

Are there other ways to express this, though, like using config?

Reply:
Yeah, the default is here for when you're in the `make bldr-shell`. I'll add instructions to BUILDER_DEV to copy the key out of 1Password and put it in this directory.

It's configurable in plans - I actually just pushed one more commit because it wasn't being done properly